js/main.js

```diff
@@ -0,0 +1,73 @@
+function validarNoVacio(input){
+    if (input.length===0){
+        return 'Este campo no puede estar vacío';
+    }
+    return '';    
+}
+
+function validarMax50Caracteres(input){
+    if (input.length>50){
+        return 'Este campo debe tener menos de 50 caracteres';
+    }
+    return '';
+}
+
+function validarMax250Caracteres(input){
+    if (input.length>250){
+        return 'Este campo debe tener menos de 250 caracteres';
+    }
+    return '';
+}
+
+function validarSoloLetras(input){
+    const contieneSoloLetras = /^[A-z]+$/.test(input);
+    if(!contieneSoloLetras){
+        return 'Este campo debe contener solo letras';
+    }
+    return '';
+}
+
+function validarNombre(nombre){
+    if (validarNoVacio(nombre)!==''){
+        return validarNoVacio(nombre);
+    }
+    if (validarMax50Caracteres(nombre)!==''){
+        return validarMax50Caracteres(nombre);
+    }    
+    if (validarSoloLetras(nombre)!==''){
+        return validarSoloLetras(nombre);
+    }
+    return 'éxito';     
+}
+
+function validarCiudad(ciudad){
+    if (validarNoVacio(ciudad)!==''){
+        return validarNoVacio(ciudad);
+    }
+    return 'éxito';  
+}
+
+function validarDescripcionRegalo(descrpcionRegalo){
```

Comment:
```suggestion
function validarDescripcionRegalo(descripcionRegalo){
```

una boludés, pero hay un typo acá